CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# 2.0.0
+
+## Added
+
+* Added `CHANGELOG.md` and `MODULE_CHANGELOG.md` for version control.
+
+## Changed
+
+* Updated `appveyor.yml` to use latest pattern from `Whiskey`.
+* Updated the `Version`, `Zip`, `CopyFile` and `Pester` task in `whiskey.yml`.

```

Comment:
Change to:

```yaml
* Updated the whiskey.yml file to use AppVeyor's deployments by default, instead of publishing with Whiskey:
  * All `*.*` branches now create builds with an `alpha` prerelease id.
  * The `develop` branch creates builds with an `rc` prerelease id.
  * Update the AppVeyor build with the version number created by Whiskey.
  * Module ZIP file now has the version number in its file name.
  * Module is now published to the .output directory by default (instead of to a gallery). This package can be published to the PowerShell Gallery with NuGet.
  * The ZIP and .nupkg files are uploaded as artificats to AppVeyor so they can be deployed with AppVeyor. It is assumed you have a GitHub deployment named `GitHub` and a PowerShell Gallery deployment named `PowerShellGallery`.
  * CHANGELOG.md, README.md, LICENSE, and NOTICE files are now included in the module when packaging.
```